### PR TITLE
cromwell-client Scala 2.13 for Agora and Rawls

### PR DESCRIPTION
**Pull request by aednichols**
- Upgraded SBT
- Updated `.gitignore` with detritus produced by new SBT version
- Fail explicitly when missing Artifactory credentials
- Cross-publish to 2.12 and 2.13

Testing: 

Ran `+publish` on commit `d334ac0` and artifacts appeared as expected:
 
https://broadinstitute.jfrog.io/artifactory/libs-release-local/org/broadinstitute/cromwell/cromwell-client_2.12/0.1-d334ac0b6-SNAP/
https://broadinstitute.jfrog.io/artifactory/libs-release-local/org/broadinstitute/cromwell/cromwell-client_2.13/0.1-d334ac0b6-SNAP/

### CODE REVIEW

**Review by mcovarr — APPROVED** [review]
If Travis approves 🥳

File: .gitignore [hunk 1]
```diff
@@ -52,3 +52,4 @@ tesk_application.conf
 **/venv/
 exome_germline_single_sample_v1.3/
 **/*.pyc
+/codegen_java/.bsp/sbt.json
```
Comment (mcovarr) [inline .gitignore:55]: what is this?

Comment (aednichols) [reply .gitignore:55]: https://www.scala-lang.org/blog/2020/10/27/bsp-in-sbt.html

Comment (aednichols) [reply .gitignore:55]: "Build Server Protocol"

**Comment by aednichols** [comment]
Oops, it looks like actually running codegen does not happen automatically, so the example artifacts are empty and non-viable:

cromwell-client_2.12-0.1-d334ac0b6-SNAP.jar                 17-Dec-2020 19:02  321 bytes

Will try to sort that out, shouldn't affect the contents of this PR though.

**Review by evanbernstein — APPROVED** [review]
Everything seems fine, but I'm certainly not catching anything that might be wrong/missing.

**Comment by aednichols** [comment]
Worked around by forcing Agora to use 2.12 instead: https://github.com/broadinstitute/agora/compare/aen_scala_2_13?expand=1

**Pull request closed without merging**